WordPress/src/main/java/org/wordpress/android/models/ReaderTagType.java

```diff
@@ -4,24 +4,20 @@
     FOLLOWED,
     DEFAULT,
     BOOKMARKED,
-    RECOMMENDED,
     CUSTOM_LIST,
     SEARCH,
     INTERESTS;
 
     private static final int INT_DEFAULT = 0;
     private static final int INT_FOLLOWED = 1;
-    private static final int INT_RECOMMENDED = 2;
-    private static final int INT_CUSTOM_LIST = 3;
-    private static final int INT_SEARCH = 4;
-    private static final int INT_BOOKMARKED = 5;
-    private static final int INT_INTERESTS = 6;
+    private static final int INT_CUSTOM_LIST = 2;
```

Comment:
⚠️ I believe we can't just change the values unless we do a full db migration. Unless I'm wrong, these values are used in the ReaderPostTable. I'd suggest simply leaving a comment - something like "// 2 - we used to use 2 for recommended tag - don't reuse it". Wdyt?

Reply:
🤦‍♀️, thanks for checking this. I'll shortly push a commit with the fix.

Reply:
No worries ;). Thanks for the fix.